task/thermal/src/bsp/medusa_a.rs

```diff
@@ -0,0 +1,104 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! BSP for Medusa
+
+use crate::control::{
+    FanControl, Fans, InputChannel, PidConfig, TemperatureSensor,
+};
+use task_sensor_api::SensorId;
+use userlib::TaskId;
+
+include!(concat!(env!("OUT_DIR"), "/i2c_config.rs"));
+
+////////////////////////////////////////////////////////////////////////////////
+// Constants!
+
+// Air temperature sensors, which aren't used in the control loop
+const NUM_TEMPERATURE_SENSORS: usize = 0;
+
+// Temperature inputs (I2C devices), which are used in the control loop.
+pub const NUM_TEMPERATURE_INPUTS: usize = 0;
+
+// External temperature inputs, which are provided to the task over IPC
+// In practice, these are our transceivers.
+pub const NUM_DYNAMIC_TEMPERATURE_INPUTS: usize =
+    drv_transceivers_api::NUM_PORTS as usize;
+
+// Number of individual fans - Medusa has none!
+pub const NUM_FANS: usize = 0;
+
+// Run the PID loop on startup
+pub const USE_CONTROLLER: bool = false;
```

Comment:
for some reason this code is extremely funny to me

not a suggestion, all the zeroes just made me giggle. carry on!

Reply:
yeah, its weird! This is one of the ways this board is breaking our usual mold. Definitely makes one ponder how modularize certain pieces are split apart.